GOVERNANCE.md

```diff
@@ -56,6 +56,18 @@ For the current list of Collaborators, see the project
 A guide for Collaborators is maintained in
 [COLLABORATOR_GUIDE.md](./COLLABORATOR_GUIDE.md).
 
+Typical activities of a Collaborator include:
+
+* contributing code and documentation changes to improve the project
+* merging pull requests
+* reviewing and commenting on issues and pull requests
+* participation in working groups
+* helping users and novice contributors
```

Comment:
Would like to see these reordered a bit by priority, with merging pull requests being the last in the list.. perhaps...

* Helping users and novice contributors
* Contributing code and documentation changes...
* Reviewing and commenting on issues and pull requests
* Participation in working groups
* Merging pull requests